### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by studioromeo**
Hey 👋 

I've updated the dependencies to their latest available versions.

Theres two main bumps here:
- PHPUnit 4.x -> 7.5
- php-cs-fixer 1.x -> 2.x

I've tried to port as much of the cs fixer config over but I noticed that it's reordered the comparison operators in some places. I hope this is ok but let me know if you'd like anything changed.

### CODE REVIEW

**Comment by studioromeo** [comment]
Heya. Sorry to bump this but I'm wondering as it's been a while if you've had any thoughts on this PR? 

I noticed #66 has been raised since I raised this one so I've adjusted this PR to meet the discussions being held in that one.

Thanks

**Comment by pyrech** [comment]
Hello @studioromeo. Sorry I completely forgot your PR :confused: Thanks a lot for it!

I just merged #66 so could please rebase your branch on top of master and resolve the conflict? :pray: Thanks

File: tests/ChangelogsPluginTest.php [hunk 1]
```diff
@@ -98,8 +99,6 @@ public function test_it_is_registered_and_activated()
         $this->addComposerPlugin($plugin);
 
         $this->assertSame([$plugin], $this->composer->getPluginManager()->getPlugins());
-        $this->assertAttributeInstanceOf('Composer\IO\IOInterface', 'io', $plugin);
-        $this->assertAttributeInstanceOf('Pyrech\ComposerChangelogs\Outputter', 'outputter', $plugin);
```
Comment (pyrech) [inline tests/ChangelogsPluginTest.php]: why did these 2 asserts disapear?

Comment (studioromeo) [reply tests/ChangelogsPluginTest.php]: Ah we may be able to add them back in again but essentially in PHPUnit 8.x `assertAttributeInstanceOf` is deprecated. Apparently the author felt that allowing tests that check the attribute value is bad practice and so removed them https://github.com/sebastianbergmann/phpunit/issues/3339

I figured here theres little to be gained by adding getters for `io` & `outputter` but can do that if you like?

Comment (pyrech) [reply tests/ChangelogsPluginTest.php]: I think I added those asserts to ensure the plugin is always activated correctly (as some of its dependencies depends from Composer) : https://github.com/pyrech/composer-changelogs/blob/master/src/ChangelogsPlugin.php#L59

But anyway, we can probably drop them :wink:

**Comment by pyrech** [comment]
Sorry @studioromeo for not taking enough time to address your comment and merge your PR. In the meantime, #68 was just merged so I guess we can now close this PR. 

Again sorry and do not hesitate if you think we miss something

**Pull request closed without merging**